OrbitGl/Action.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORIBIT_GL_ACTION_H_
+#define ORIBIT_GL_ACTION_H_
+
+// Actions are executed by MainThreadExecutor
+// The Action is an abstract class which can
+// be executed. See below for basic implementations for
+// this class: VoidAction and UnaryAction.
+class Action {
+ public:
+  Action() = default;
+  virtual ~Action() = default;
+
+ public:
```

Comment:
is the second "public:" intentional here?

Reply:
leftover after simplification - removed.